### PR TITLE
Adds better messaging. 

### PR DESCRIPTION
**Pull request by keeganbrown**
# Why?
- There was a reference error that resulted in a 500, instead of a 404. #25 

# What changed?
- Fixed the error, and added some messaging to help diagnose if there are other issues in the same vein.

### CODE REVIEW

File: lib/crbot.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ class CrBot {
   onIncomingCodeReview(req, res, next) {
     let githubPRParams = this.getGitHubPRParams(req.body.text);
     let newCR;
-    if (!githubPRParams) { return rs.status(404).end(); }
+    if (!githubPRParams) { return res.status(404).end(`Beep:Boop. I could not find your repo. Are you sure it is accessible to me? Please double check your URL: ${req.body.text}`); }
```
Comment (greg5green) [inline lib/crbot.js]: Could you spread this out into a few different lines?

Comment (keeganbrown) [inline lib/crbot.js]: Yah dood!

Comment (greg5green) [inline lib/crbot.js]: Sweet. :shipit:

**Comment by jahammo2** [comment]
I also agree with Greg, :shipit:

**Pull request merged**